lib/addons/src/make-decorator.js

```diff
@@ -19,6 +19,10 @@ export const makeDecorator = ({
   const decorator = options => (getStory, context) => {
     const parameters = context.parameters && context.parameters[parameterName];
 
+    if (parameters && parameters.skip) {
```

Comment:
Note I added a new `{ skip: true }` behaviour to `makeDecorator()`. *NOTE THAT THIS WILL APPLY TO ALL ADDONS* so let me know if it should be opt-in. I think it's good to have a consistent API that users will expect to be there.

Also, LMK if `skip` is a bad name (`disable` could be a good option as well)

Reply:
I like `disable` better personally

Reply:
Let's leave it for 24 hours to let other weigh in. I am more than happy to change it but will wait.

Reply:
Prefer `disable` as well. Would it be more intuitive however to be able to do something like `{ backgrounds: false }` instead of nesting another object? (Just throwing it out there, not sure how it will affect the API)

Reply:
Oh, good idea, I think I prefer that too. @ndelangen / @shilman ?

Reply:
+1 for `false`

Reply:
Fair point.

The current behaviour treats falsey things (both `false` and `null`) as if you didn't set the parameter at that level of the stack.

So if you pass `{ backgrounds: false }` you will either inherit the backgrounds parameter, or else have the parameter unset--which in the case of backgrounds would lead to the addon not applying--however it is a little case-by-case (i.e. up to the addon) what will happen. 

Although it doesn't make sense to set the parameter to `null`, I see the point that it could easily lead to bugs when compared to the default which is to set it to `undefined`. 

So I guess the consensus is to go with `{ disable: true }`. I will run with this change. We can always add the `false` API later.